### ISSUE TITLE
Training Loss -0.99550, Validation Loss -0.99271

### ISSUE THREAD

**Issue by Tomhouxin** [xuduo35/unet_mxnet2ncnn#6]
when i run trainunet, I get train_loss and validat_loss, It doesn't feel right

New training ...
 Best model at Epoch 1
Epoch 0: Moving Training Loss -0.98716, Validation Loss -0.98693
......
Epoch 23: Moving Training Loss -0.99521, Validation Loss -0.99272
Epoch 24: Moving Training Loss -0.99550, Validation Loss -0.99271
.....

**Comment by xuduo35** [comment]
Training Loss should be -0.97~0.90, Validation Loss also.

**Comment by Tomhouxin** [comment]
Why is it negative ?
Is there any other network model that can achieve real time?

**Comment by xuduo35** [comment]
Because the code use NOT dice_coef as loss. Don't know other real time network, you can google it.
